src/app.py

```diff
@@ -2721,7 +2719,7 @@ def get_prov_info():
     # Token is not required, but if an invalid token is provided,
     # we need to tell the client with a 401 error
     validate_token_if_auth_header_exists(request)
-
+    organ_types_dict = schema_manager.get_organ_types()
```

Comment:
Looks like the `organ_types_dict` is assigned twice with the same call: https://github.com/hubmapconsortium/entity-api/blob/95b243e3da97d3c9138507ca598e64acb056cf28/src/app.py#L2722-L2728

Just keep one.